### PR TITLE
[THEMES] Add ukrainian translations

### PR DESCRIPTION
**Pull request by ConLenov**
* Add ukrainian translation

* Add ukrainian translation

### CODE REVIEW

**Comment by binarymaster** [comment]
> 10 commits into `reactos:releases/0.4.13`

Incorrect target branch, please reopen into `reactos:master`.

**Comment by cernodile** [comment]
If the inclusion of first commits were a mistake, I'd advise you learn about how branches work in git.

File: media/themes/Modern/modern.msstyles/modern.rc [hunk 1]
```diff
@@ -444,3 +444,6 @@ DARK_USERTILEBACKGROUND_BMP BITMAP  "bitmaps/Dark/DARK_USERTILEBACKGROUND.bmp"
 #ifdef LANGUAGE_ZH_CN
     #include "lang/zh-CN.rc"
 #endif
+#ifdef LANGUAGE_UA_UA
```
Comment (cernodile) [inline media/themes/Modern/modern.msstyles/modern.rc]: There is no language such as "UA_UA", did you mean "UK_UA". (This applies to every translation made)

**Comment by binarymaster** [comment]
Yeah you need to clean up unrelated commits.

**Comment by cernodile** [comment]
Can you please group your commits in a single batch, rather than do a separate commit each time you change a file. This clogs up email of people participating in this PR. Thanks. Also, you didn't fully address the issue, the file name should be uk-UA.rc as well.

**Pull request closed without merging**